app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,12 @@
+<header class="navbar"> 
+  <ul class="links">
+    <li><%= link_to "Haxxor-News", "#" %></li> 
```

Comment:
It can be helpful to put `#TODO` inside the href for links that don't yet have destinations.